src/ASM/LR/ASMu3D.C

```diff
@@ -2282,3 +2282,46 @@ void ASMu3D::getElmConnectivities (IntMat& neigh) const
     }
   }
 }
+
+
+void ASMu3D::getBoundaryElms (int lIndex, int orient, IntVec& elms) const
+{
+  LR::parameterEdge edge;
+  switch (lIndex)
+  {
+  case 1: edge = LR::WEST;   break;
+  case 2: edge = LR::EAST;   break;
+  case 3: edge = LR::SOUTH;  break;
+  case 4: edge = LR::NORTH;  break;
+  case 5: edge = LR::BOTTOM; break;
+  case 6: edge = LR::TOP;    break;
+  default:edge = LR::NONE;
+  }
+
+  std::vector<LR::Element*> elements;
+  this->getBasis(1)->getEdgeElements(elements, edge);
+
+  int dir = (lIndex - 1) / 2;
+  int u = dir == 0 ? 1 : 0;
+  int v = 1 + (dir != 2 ? 1 : 0);
```

Comment:
Move these three inside the lambda scope?

Reply:
done.